test/Npgsql.CrateDb.Tests/CrateDbIntegrationTest.cs

```diff
@@ -0,0 +1,174 @@
+                    @short_field, @integer_field, @long_field, @float_field, @double_field, @object_field, 
+                    @timestamp_field, @ip_field, @geo_point_field, @geo_shape_field)";
+
+                cmd.Parameters.AddWithValue("@id", 1);
+                cmd.Parameters.AddWithValue("@string_field", "Youri");
+                cmd.Parameters.AddWithValue("@boolean_field", true);
+                cmd.Parameters.AddWithValue("@byte_field", (byte)120);
+                cmd.Parameters.AddWithValue("@byte_field", DBNull.Value);
+                cmd.Parameters.AddWithValue("@short_field", (short)1000);
+                cmd.Parameters.AddWithValue("@integer_field", 1200000);
+                cmd.Parameters.AddWithValue("@long_field", 120000000000L);
+                cmd.Parameters.AddWithValue("@float_field", 1.4f);
+                cmd.Parameters.AddWithValue("@double_field", 3.456789);
+
+                cmd.Parameters.AddWithValue("@object_field", NpgsqlTypes.NpgsqlDbType.Json, "{ \"inner\": \"Zoon\" }");
+                //cmd.Parameters.AddWithValue("@object_field", NpgsqlTypes.NpgsqlDbType.Json, new { inner = "Zoon" });
+
+                cmd.Parameters.AddWithValue("@timestamp_field", new DateTime(2018, 10, 11).ToUniversalTime());
+                cmd.Parameters.AddWithValue("@ip_field", "127.0.0.1");
+                cmd.Parameters.AddWithValue("@geo_point_field", new double[] { 9.7419021d, 47.4048045d });
+                cmd.Parameters.AddWithValue("@geo_shape_field", "POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))");
+                cmd.ExecuteNonQuery();
+
+
+
+                cmd.CommandText = "refresh table test";
+                cmd.ExecuteNonQuery();
+            }
+        }
+
+        protected void TearDownTestTable()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = con.CreateCommand())
+            {
+                cmd.CommandText = @"drop table if exists test";
+                cmd.ExecuteNonQuery();
+            }
+        }
+
+        protected void CreateArrayTestTable()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = con.CreateCommand())
+            {
+                cmd.CommandText = @"create table if not exists arrayTest (
+                    id integer primary key,
+                    str_array array(string),
+                    bool_array array(boolean),
+                    byte_array array(byte),
+                    short_array array(short),
+                    integer_array array(integer),
+                    long_array array(long),
+                    float_array array(float),
+                    double_array array(double),
+                    timestamp_array array(timestamp),
+                    obj_array array(object),
+                    geo_shape_array array(geo_shape)
+                    ) clustered by (id) into 1 shards with (number_of_replicas=0)";
```

Comment:
The default column policy will change in the upcoming version of CrateDB >= 4.0 https://github.com/crate/crate/commit/e80cd3c9af630508816372043bdc9bd6edf81ab7. So set it explicitly to `dynamic` here.

```suggestion
                    ) clustered by (id) into 1 shards with (number_of_replicas=0, column_policy = 'dynamic')";
```